### PR TITLE
Getenv refactor

### PR DESCRIPTION
**Pull request by vincentzed**
## Motivation

With codemod, and manually removed/added instances
Refactor the `os` usages of both ways to get env var, and remove some duplicated helpers. Also, note that we don't touch `examples` and else for simplicity.

## Checklist

- [ ] Format your code according to the [Format code with pre-commit](https://docs.sglang.ai/developer_guide/contribution_guide.html#format-code-with-pre-commit).
- [ ] Add unit tests according to the [Run and add unit tests](https://docs.sglang.ai/developer_guide/contribution_guide.html#run-and-add-unit-tests).
- [ ] Update documentation according to [Write documentations](https://docs.sglang.ai/developer_guide/contribution_guide.html#write-documentations).
- [ ] Provide accuracy and speed benchmark results according to [Test the accuracy](https://docs.sglang.ai/developer_guide/contribution_guide.html#test-the-accuracy) and [Benchmark the speed](https://docs.sglang.ai/developer_guide/contribution_guide.html#benchmark-the-speed).

### CODE REVIEW

**Comment by vincentzed** [comment]
@JustinLin610 Can you help look? Thx!

File: python/sglang/test/test_deepep_utils.py [hunk 1]
```diff
@@ -8,13 +6,17 @@
 import torch
 import torch.distributed as dist
 
+from sglang.srt.utils import get_int_env_var
+
+# Copy from deepseek-ai/DeepEP/tests/test_utils.py
```
Comment (vincentzed) [inline python/sglang/test/test_deepep_utils.py]:
```suggestion
```

**Comment by JustinTong0323** [comment]
fix lint error plz~

**Comment by vincentzed** [comment]
updated

**Comment by JustinTong0323** [comment]
Duplicate with https://github.com/sgl-project/sglang/pull/10479 but really thanks for your work!
